test/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const CODE_SERVER_ADDRESS = process.env.CODE_SERVER_ADDRESS || "http://localhost:8080"
+export const PASSWORD = process.env.PASSWORD || "e45432jklfdsab"
+export const STORAGE = process.env.STORAGE || ""
```

Comment:
I think here we'll need to tweak this a bit.

The reason is that at the top of the global setup file we import this file which causes `STORAGE` to get set to `process.env.STORAGE` (which will be undefined at that point so we'll end up setting it to a blank string).

Then later we set the storage to `process.env.STORAGE` but this won't update the `STORAGE` constant since we already set it to a blank string.

Some ideas:
1. Make this a `let` and set it directly instead of `process.env.STORAGE`
2. Make this a function so it grabs the current `process.env.STORAGE`

Reply:
Or, wait, am I wrong about how this works? If I was right then the `JSON.parse` should have errored since it can't parse a blank string. But the tests passed just fine.

Reply:
Oh maybe it's Jest clearing the module cache before/after each test or something? In which case this is completely fine.

Reply:
Yup confirmed that's what's going on.